implementations/kotlin-kmp-json-schema-validator/src/main/kotlin/Responses.kt

```diff
@@ -17,6 +17,9 @@ class Implementation(
     val homepage: String,
     val issues: String,
     val source: String,
+    val os: String,
+    val os_version: String,
```

Comment:
To keep the naming match kotlin naming convention you can use `kotlinx.serialization.SerialName` annotation.

`@SerialName("os_version") val osVersion: String`